<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_widget</class>
 <widget class="QWidget" name="Camera_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="selectCamera">
      <property name="text">
       <string>SelectCamera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OptionsCamera">
      <property name="text">
       <string>OptionsCamera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CaptureImage">
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>1431</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCameraViewfinder" name="showCamera" native="true">
      <property name="maximumSize">
       <size>
        <width>712</width>
        <height>599</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imageViewer">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
